src/shell/commands/debugger.cpp

```diff
@@ -129,6 +130,29 @@ bool mlog_dump(command_executor *e, shell_context *sc, arguments args)
                        << pegasus::utils::c_escape_string(hash_key, sc->escape_all) << "\" : \""
                        << pegasus::utils::c_escape_string(sort_key, sc->escape_all) << "\" => "
                        << update.increment << std::endl;
+                } else if (msg->local_rpc_code == ::dsn::apps::RPC_RRDB_RRDB_CHECK_AND_SET) {
```

Comment:
So many `if..else..`, you can do some refactor to use `switch..case..`

Reply:
no, you cannot switch/case on a task_code. Internally, task_code is not a constant integer. The code number is generated in runtime rather than compile-time.